schema-next.yaml

```
@@ -2,6 +2,18 @@ file_format: 1.1.0
 schema_url: https://opentelemetry.io/schemas/next
 versions:
   next:
+    metrics:
```

```suggestion
    all:
```

it does not matter where the attribute was used: spans or metrics or resources, it should be renamed everywhere (and yes, this file schema is not perfect)